oaknational/Oak-Web-Application
# [CUR-370] fix: update curriculum sitemap

## Description

Music year: 2004

- Fixed typo: changed `lastmid` to `lastmod` in sitemap

## How to test

1. Go to https://deploy-preview-2162--oak-web-application.netlify.thenational.academy/teachers/curriculum/sitemap.xml
2. View the XML as source
3. You should see several items with the format: `<url><loc>...</loc><lastmod>...</lastmod></url>`
